pkg/parser/parser.y

```diff
@@ -1866,6 +1866,17 @@ DirectResourceGroupOption:
 	{
 		$$ = &ast.ResourceGroupOption{Tp: ast.ResourceRURate, UintValue: $3.(uint64)}
 	}
+|	"RU_PER_SEC" EqOpt stringLit
```

Comment:
Can we directly use "UNLIMITED"? like `WatchDurationOption`

Reply:
`WatchDurationOption` still needs to be checked by the `strings.ToLower`, which is maybe no different than now. 

https://github.com/pingcap/tidb/blob/8f0c053a38b4b99d00d5eb6f466c783d11a1f899/pkg/parser/parser.y#L1834-L1849

Reply:
good catch, using `UNLIMITED` now.